scripts/st2bootstrap-el7.sh

```diff
@@ -0,0 +1,183 @@
+#!/bin/bash
+
+set -eux
+
+# Note that default SELINUX policies for RHEL7 differ with CentOS7. CentOS7 is more permissive by default
+# Note that depending on distro assembly/settings you may need more rules to change
+# Apply these changes OR disable selinux in /etc/selinux/config (manually)
+adjust_selinux_policies() {
+  if getenforce | grep -q 'Enforcing'; then
+    # Allow rabbitmq to use '25672' port, otherwise it will fail to start
+    sudo semanage port --list | grep -q 25672 || sudo semanage port -a -t amqp_port_t -p tcp 25672
+
+    # Allow network access for nginx
+    sudo setsebool -P httpd_can_network_connect 1
+  fi
+}
+
+install_st2_dependencies() {
+  sudo yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+  sudo yum -y install mongodb-server rabbitmq-server
+  sudo systemctl start mongod rabbitmq-server
+}
+
+install_st2() {
+  curl -s https://packagecloud.io/install/repositories/StackStorm/staging-stable/script.rpm.sh | sudo bash
+  sudo yum -y install st2
+  sudo st2ctl reload
+  sudo st2ctl start
+}
+
+configure_st2_user() {
+  # Create an SSH system user (default `stanley` user may be already created)
+  if (! id stanley 2>/dev/null); then
+    sudo useradd stanley
+  fi
+  
+  sudo mkdir -p /home/stanley/.ssh
+  sudo chmod 0700 /home/stanley/.ssh
+  
+  # On StackStorm host, generate ssh keys
+  sudo ssh-keygen -f /home/stanley/.ssh/stanley_rsa -P ""
+  
+  # Authorize key-base acces
+  sudo cat /home/stanley/.ssh/stanley_rsa.pub >> /home/stanley/.ssh/authorized_keys
```

Comment:
I ran ./st2bootstrap-el7.sh (note without sudo)
+ sudo cat /home/stanley/.ssh/stanley_rsa.pub
./st2bootstrap-el7.sh: line 44: /home/stanley/.ssh/authorized_keys: Permission denied

Fails. 

http://askubuntu.com/questions/351403/permission-denied-while-making-a-file-without-asking-for-password
This works:
sudo cat /home/stanley/.ssh/stanley_rsa.pub | sudo tee -a /home/stanley/.ssh/stanley_rsa.pub